### PR TITLE
Re-introduce comments about frustum culling

### LINKED ISSUE #8490: Document frustum culling in 2d examples
## How can Bevy's documentation be improved?

Since #7885, bevy again runs frustum culling on 2d stuff. It's a good reason to revert #7436 (or at least re-introduce frustum language to the example comment)

### PR DESCRIPTION
**Pull request by hate**
# Objective

Frustum culling for 2D components has been enabled since #7885,
Fixes #8490 

## Solution

Re-introduced the comments about frustum culling in the many_animated_sprites.rs and many_sprites.rs examples.

### CODE REVIEW

File: examples/stress_tests/many_animated_sprites.rs [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
 //!
 //! It sets up many animated sprites in different sizes and rotations,
-//! and at different scales in the world, and moves the camera over them.
+//! and at different scales in the world,
+//! and moves the camera over them to see how well frustum culling works.
 //!
-//! Having sprites out of the camera's field of view should also help stress
-//! test any future potential 2d frustum culling implementation.
+//! To measure performance realistically, be sure to run this in release mode.
+//! `cargo run --example many_animated_sprites --release`
```
Comment (nicopap) [inline examples/stress_tests/many_animated_sprites.rs:8]: This paragraph not needed anymore, it is now automatically printed in the console when running the example. (also need to remove the empty `//!` line on top)

```suggestion
```

File: examples/stress_tests/many_animated_sprites.rs [hunk 2]
```diff
@@ -1,10 +1,9 @@
 //! Renders a lot of animated sprites to allow performance testing.
 //!
 //! It sets up many animated sprites in different sizes and rotations,
-//! and at different scales in the world, and moves the camera over them.
+//! and at different scales in the world,
+//! and moves the camera over them to see how well frustum culling works.
 //!
```
Comment (nicopap) [inline examples/stress_tests/many_animated_sprites.rs:6]: Trailing empty doc line here.

```suggestion
```

File: examples/stress_tests/many_animated_sprites.rs [hunk 3]
```diff
@@ -1,10 +1,9 @@
 //! Renders a lot of animated sprites to allow performance testing.
 //!
 //! It sets up many animated sprites in different sizes and rotations,
-//! and at different scales in the world, and moves the camera over them.
+//! and at different scales in the world,
+//! and moves the camera over them to see how well frustum culling works.
```
Comment (nicopap) [inline examples/stress_tests/many_animated_sprites.rs:5]: Nit: The sentence as a whole here is

> It sets up many animated sprites in different sizes and rotations, and at different scales in the world, and moves the camera over them to see how well frustum culling works.

I think there are too many "and". What about the following?

> This example sets up many animated sprites in different sizes, rotations, and scales in the world. It also moves the camera over them to see how well frustum culling works.

Comment (hate) [reply examples/stress_tests/many_animated_sprites.rs:5]: You are right, that does flow better, thank you

**Review by nicopap — APPROVED** [review]
LGTM

File: examples/stress_tests/many_sprites.rs [hunk 4]
```diff
@@ -1,11 +1,8 @@
 //! Renders a lot of sprites to allow performance testing.
 //! See <https://github.com/bevyengine/bevy/pull/1492>
 //!
-//! It sets up many animated sprites in different sizes and rotations,
-//! and at different scales in the world, and moves the camera over them.
-//!
-//! Having sprites out of the camera's field of view should also help stress
-//! test any future potential 2d frustum culling implementation.
+//! This exampple sets up many sprites in different sizes, rotations, and scales in the world.
```
Comment (mockersf) [inline examples/stress_tests/many_sprites.rs:4]:
```suggestion
//! This example sets up many sprites in different sizes, rotations, and scales in the world.
```

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**